tests/core/coverage/lcov_coverage_test.go

```diff
@@ -130,7 +131,7 @@ func TestLib(t *testing.T) {
 }
 
 func TestTool(t *testing.T) {
-	err := exec.Command("Tool").Run()
+	err := exec.Command("." + string(filepath.Separator) + "Tool").Run()
```

Comment:
Is this change due to the recent security fix?

Reply:
I think this was changed when 1.19 was released: https://tip.golang.org/doc/go1.19#os-exec-path

Reply:
I pulled this out into https://github.com/bazelbuild/rules_go/pull/3392.

Reply:
done, ran into some difficulties so I had to force push